languages/python/exercises/concept/strings/.docs/after.md

```diff
@@ -0,0 +1,17 @@
+
+In this lesson, we learned:
+ - Python has a `str` type.
+ - How to find items in a string.
+ - How to manipulate strings to create new strings.
+ - Where to find documentation about string methods.
+
+## More about strings
+
+It's useful to become familiar with Python's string instance methods, formatting, and literal syntax. For a deep dive on what information a string is encoding (or, "how does the computer know how to translate zeroes and ones into letters?"), [this blog post is enduringly helpful](https://www.joelonsoftware.com/2003/10/08/the-absolute-minimum-every-software-developer-absolutely-positively-must-know-about-unicode-and-character-sets-no-excuses/).
+
+### More about Python logging
+
+Python has a useful [built-in logging library](https://docs.python.org/3.8/library/logging.html), aptly-named `logging`. It handles cases similar
+to the ones you're working on, and more. It's very often used in projects, and is worthwhile to know.
+
+https://docs.python.org/3.8/library/logging.html
```

Comment:
Links included without markdown link styling will not always render as live links in the resulting HTML, probably best to put this in a link tag.

Reply:
Or, now that I notice it's a duplicate of a link on line 14, just delete it or complete moving the link down here as an indirect one.